pkg/features/features.go

```diff
@@ -140,7 +141,7 @@ var AllFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	RemoveAPIServerProxyLegacyPort:           {Default: true, PreRelease: featuregate.Beta},
 	IstioTLSTermination:                      {Default: false, PreRelease: featuregate.Alpha},
 	CloudProfileCapabilities:                 {Default: false, PreRelease: featuregate.Alpha},
-	DoNotCopyBackupCredentials:               {Default: false, PreRelease: featuregate.Alpha},
+	DoNotCopyBackupCredentials:               {Default: true, PreRelease: featuregate.Beta},
```

Comment:
We could now drop https://github.com/gardener/gardener/blob/452451fb04c16e415f3908bddbd50da6b06e3206/example/gardener-local/gardenlet/values.yaml#L42.